src/app/views/functions/_functions.html.erb

```diff
@@ -1,3 +1,28 @@
+<% if current_user %>
```

Comment:
if you removed this condition you can have this to any user, so you can have this list public for anyone, this would be nice, as you can go to a user and see what he saved/starred to discover another user.

And review the controller to make sure it works with the user passed in parameter instead of current_user